.github/workflows/main.yml

```diff
@@ -117,6 +126,143 @@ jobs:
+          echo "  edge:"                >> ~/.fablab/bindings.yml
+          echo "    username: admin"    >> ~/.fablab/bindings.yml
+          echo "    password: admin"    >> ~/.fablab/bindings.yml
+          echo "  influxdb:"            >> ~/.fablab/bindings.yml
+          echo "    username: fablab"   >> ~/.fablab/bindings.yml
+          echo "    password: fablab"   >> ~/.fablab/bindings.yml
+
+      - name: Test Ziti Command
+        env:
+          ZITI_ROOT: release/amd64/linux/
+          ZITI_VERSION: ${{ steps.get_version.outputs.ziti-version }}
+        run: |
+          release/amd64/linux/bin/simple-transfer create
+          release/amd64/linux/bin/simple-transfer up
+          release/amd64/linux/bin/simple-transfer run
+
+      - name: Test Ziti Command Teardown
+        if: always()
+        env:
+            ZITI_ROOT: release/amd64/linux/
+        run: |
+          release/amd64/linux/bin/simple-transfer dispose
+          release/amd64/linux/bin/simple-transfer clean
+
+      # END linux-build-steps
+
+  publish:
+    name: Publish Binaries and Docker Image
+    runs-on: ubuntu-20.04
+    needs: [ mac-os-build, windows-build ]
+    outputs:
+      ziti-version: ${{ steps.get_version.outputs.ziti-version }}
+    steps:
+      - name: Git Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Install Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: '1.18.x'
+
+      - name: Install Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.7'
+
+      - name: Install Ziti CI
+        uses: netfoundry/ziti-ci@v1
+
+      - name: Build and Test
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          ziti_ci_gpg_key: ${{ secrets.ZITI_CI_GPG_KEY }}
+          ziti_ci_gpg_key_id: ${{ secrets.ZITI_CI_GPG_KEY_ID }}
+        run: |
+          sudo apt-get update
+          sudo apt-get -yq install gcc-arm-linux-gnueabihf g++-arm-linux-gnueabihf gcc-aarch64-linux-gnu
+          $(go env GOPATH)/bin/ziti-ci configure-git
+          $(go env GOPATH)/bin/ziti-ci generate-build-info common/version/info_generated.go version
+          go install github.com/mitchellh/gox@latest
+          $(go env GOPATH)/bin/gox -cgo -os=linux -arch=amd64 -output=$GOX_OUTPUT ./...
+          CC=arm-linux-gnueabihf-gcc $(go env GOPATH)/bin/gox -cgo -os=linux -arch=arm -output=$GOX_OUTPUT ./...
+          CC=aarch64-linux-gnu-gcc   $(go env GOPATH)/bin/gox -cgo -os=linux -arch=arm64 -output=$GOX_OUTPUT ./...
+
+          mkdir -p test/amd64/linux/bin
+          go build -tags=all,tests -o test/amd64/linux/bin ./...
+          CC=arm-linux-gnueabihf-gcc $(go env GOPATH)/bin/gox -cgo -os=linux -arch=arm -output=$GOX_TEST_OUTPUT -tags=all ./...
+          CC=aarch64-linux-gnu-gcc   $(go env GOPATH)/bin/gox -cgo -os=linux -arch=arm64 -output=$GOX_TEST_OUTPUT -tags=all ./...
+
```

Comment:
Don't need upload, we're building them here

Reply:
Artifacts from a bad copy/paste that I glossed over